### PR TITLE
Subscribe / presence stateless methods

### PR DESCRIPTION
**Pull request by parfeon**
feat: subscribe / presence stateless methods

Add stateless methods for subscribe / presence API.

### CODE REVIEW

File: Sources/PubNub/Subscription/SubscriptionSession.swift [hunk 1]
```diff
@@ -33,7 +33,7 @@ public class SubscriptionSession {
 
   public let uuid = UUID()
   let longPollingSession: SessionReplaceable
-  internal(set) var configuration: SubscriptionConfiguration
+  var configuration: SubscriptionConfiguration
```
Comment (kleewho) [inline Sources/PubNub/Subscription/SubscriptionSession.swift:36]: Does it have to change?

**Review by kleewho — COMMENTED** [review]
What about `setPresenceState` utility?

File: Sources/PubNub/Subscription/SubscriptionSession.swift (hunk 1 above)
Comment (parfeon) [reply Sources/PubNub/Subscription/SubscriptionSession.swift]: @kleewho this one changed in master already.

**Comment by parfeon** [comment]
@kleewho 

> What about `setPresenceState` utility?

Swift SDK already have `setPresence`. So I'm kind of confusing in why we need to create dupe.

**Comment by CraigLn** [comment]
@parfeon I'm just trying to close out PRs that I'm attached to.  Is this PR still valid?

**Comment by parfeon** [comment]
Yep. This one still valid. Waiting when there will be decision to make release.
Will send it to draft so far.

**Comment by parfeon** [comment]
This has been done in a different way.

**Pull request closed without merging**